spring20-wvd-templates/Create and provision WVD host pool/CreateHostpoolTemplate.json

```diff
@@ -0,0 +1,701 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "nestedTemplatesLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "The base URI where artifacts required by this template are located."
+      },
+      "defaultValue": "https://catalogartifact.azureedge.net/publicartifacts/Microsoft.Hostpool-ARM-1.0.2-preview/"

```

Comment:
Same note about pointing to github master

Reply:
fix